### ISSUE TITLE
What to put in the code to scan multiple locations at time?

### ISSUE THREAD

**Issue by Bmoney99** [AHAAAAAAA/PokemonGo-Map#3230]
I'm wondering how I can get multiple locations scanned at a time on one webpage. I know I would need multiple accounts, but what do I put in the code in order to be able to do this?

**Comment by danielzt12** [comment]
use --no-server on all but one account

**Comment by hagehg** [comment]
you need to run multiple instances of runserver.py.

One of them need to be your server, run it like you usually would.

The others need to run in no-server mode, run them with the -ns argument

**Comment by Oricul** [comment]
I could be wrong on this - but I also had to setup a DB. Originally, I just specified the same DB using -D "<path_to_db>\<whatever_name>"

I now use a MySQL DB, but I don't think that's necessary unless you want multiple servers (different from instances) running.

**Comment by Bmoney99** [comment]
Thank you everyone, I'm going to attempt this in a second. I'll keep you updated, I appreciate it!

**Comment by shivamkalra** [comment]
I did not have to create mysql database..just running off pogom.db is fine for me. Will I eventually run into issues?

**Comment by Bmoney99** [comment]
@hagehg @danielzt12 @Oricul Okay I've got 3 threads so that is very helpful but trying to set up multiple lines causes one to fail even if -ns is used, do I have to have the dev build?

**Comment by Josh-G** [comment]
> I did not have to create mysql database..just running off pogom.db is fine for me. Will I eventually run into issues?
No, running multiple local instances is fine when using the default SQLite db.

> @hagehg @danielzt12 @Oricul Okay I've got 3 threads so that is very helpful but trying to set up multiple lines causes one to fail even if -ns is used, do I have to have the dev build?
It fails in what way? Do you have an error message? Doing this works fine for me on dev branch.

**Comment by Bmoney99** [comment]
@Josh-G I'm not using the dev build at the moment is that the problem?  Also, it's ignoring my second line of runserver.py. with a different account now.

**Comment by InflamedSebi** [comment]
@shivamkalra SQLite is actually a single user database. It will work with more users (each running script =1 user) as long as they don't access it the exact same time.But multiple instances could collide and one of the instances would not be able to write the data and discard it after a time out (this would show up as error in the log).

**Comment by shivamkalra** [comment]
@InflamedSebi I've 6 instances running. I do sometimes see keyerror. Do you know what kind of error it might throw in logs?

**Comment by shivamkalra** [comment]
@InflamedSebi I must also mention, each instances has like 6 threads on average.

**Comment by InflamedSebi** [comment]
@shivamkalra sadly the errormessage seems not to be consistend. Some get a "db locked" error, others a "timeout" error. I got some weird python error gibberish. But everything is gone since I switched to mysql. If errors happen rarely, i wouldn't mind. The worst thing that could happen is a single skipped scan.
